src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -73,7 +77,8 @@ public void hasPerson_personInAddressBook_returnsTrue() {
     @Test
     public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
         addressBook.addPerson(ALICE);
-        Person editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
+        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB)
+                .withAddress(VALID_ADDRESS_BOB).withUniversity(VALID_UNIVERSITY_BOB).withTags(VALID_TAG_HUSBAND)
```

Comment:
You should not change identity fields here (phone and email) because the test us for person with same identity fields (see Person.java) to check what are the identity fields